packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.11.2
+
+* Adds support to camera control button on web.
```

Comment:
"Adds support for disabling or moving the camera control button on web."

Reply:
done.